plugins/modules/synchronize.py

```diff
@@ -636,15 +633,20 @@ def _write_password_to_pipe(proc):
     out_lines = out_clean.split('\n')
     while '' in out_lines:
         out_lines.remove('')
-    if module._diff:
-        diff = {'prepared': out_clean}
-        return module.exit_json(changed=changed, msg=out_clean,
-                                rc=rc, cmd=cmdstr, stdout_lines=out_lines,
-                                diff=diff)
 
-    return module.exit_json(changed=changed, msg=out_clean,
-                            rc=rc, cmd=cmdstr, stdout_lines=out_lines)
+    result = dict(changed=changed, rc=rc, cmd=cmdstr)
+
+    if quiet:
+        result['msg'] = "OUTPUT IS HIDDEN DUE TO 'quiet=true'"
```

Comment:
I think it'd be better to just omit the output, or output a summary ("X files synchronized"). An "OUTPUT IS HIDDEN" message feels a big aggressive (the caps), and unnecessary (I told it silent, I don't need it telling me I told it to do something).

Reply:
Makes sense, the caps do seem aggressive. Thank you for the feedback.